src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/overloadmethodsdeclarationorder/InputOverloadMethodsDeclarationOrder.java

```diff
@@ -60,6 +60,13 @@ public void fooMethod()
         public void overloadMethod(String s, Boolean b, int i) // violation
         {
             //some foo code
+        };
+
+        public void overloadMethod(double d) // violation
+                                            // because there is an unnecessary semicolon at the
+                                            // end of the method
```

Comment:
where is this semicolon at the end of **this** method? Also, this issue isn't about a semicolon so I am not sure why this specific method is being added.

Reply:
sorry the explanation comment was not written properly. I have explained it properly now:

https://github.com/checkstyle/checkstyle/blob/24ab0e93ad43a656fca70f6a7fd16b9d559bafdf/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/overloadmethodsdeclarationorder/InputOverloadMethodsDeclarationOrder.java#L65-L68

There is a semicolon at the end of the above method which is separating the overloaded methods.  That's why it is causing the violation. We found this test case here: https://github.com/checkstyle/checkstyle/pull/14766#pullrequestreview-1986501881

Mr.Roman suggested to add this to the input test files: https://github.com/checkstyle/checkstyle/pull/14766#discussion_r1561053086

Reply:
This comment spanning so many lines isn't much better.

Can we not use the comment like "Violation above" or "Violation below" here?

`at the end of the above method` => `at the end of the previous method`

Reply:
done. Here's the new changes:

https://github.com/checkstyle/checkstyle/blob/922cba38624e05652b2d9cb37f13e342d0d4c03b/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/overloadmethodsdeclarationorder/InputOverloadMethodsDeclarationOrder.java#L65-L74

@rnveach